packages/block-library/src/media-text/edit.js

```diff
@@ -187,6 +191,35 @@ function MediaTextEdit( { attributes, isSelected, setAttributes } ) {
 	const onVerticalAlignmentChange = ( alignment ) => {
 		setAttributes( { verticalAlignment: alignment } );
 	};
+
+	const { imageSizes } = useSelect( ( select ) => {
+		const { getSettings } = select( 'core/block-editor' );
+		return pick( getSettings(), [ 'imageSizes' ] );
+	} );
+	const imageSizeOptions = map(
+		filter( imageSizes, ( { slug } ) =>
+			get( image, [ 'media_details', 'sizes', slug, 'source_url' ] )
```

Comment:
We kinda started adopting [conditional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) (`?.`) instead of `get()`

```suggestion
			image?.media_details?.sizes?.[ slug ]?.source_url
```